webapp/src/components/LinkedProfile/LinkedProfile.tsx

```diff
@@ -9,7 +9,7 @@ export const LinkedProfile = ({ isProfileEnabled, ...props }: Props) => {
   const redirectionProps: RedirectionProps = isProfileEnabled
     ? {
         as: 'a',
-        href: `${profileUrl}/${address}`
+        href: `${profileUrl}${locations.account(address)}`
```

Comment:
```suggestion
        href: `${profileUrl}/accounts/${address}`
```
What do you think of hardcoding the path instead of using locations? I'm a bit concerned about the location function being for locations inside of the Marketplace. If we eventually change or delete the account path in the Marketplace, we'll be be changing the path to the profile url.

Reply:
good catch! let me update it